appveyor.yml

```diff
@@ -17,28 +17,33 @@ environment:
 
   matrix:
     # for testing purpose: numpy 1.8 on py2.7, for the rest use 1.10/latest
+    # theoretically the CONDA_INSTALL_LOCN could be only two: one for 32bit, one for 64bit
+    # but using one for the right python version is hopefully making it fast due to package caching.
     - TARGET_ARCH: "x86"
       CONDA_PY: "27"
       CONDA_NPY: "18"
       PYTHON_VERSION: "2.7"
+      TEST_ALL_IMAGES: "no"
       CONDA_INSTALL_LOCN: "C:\\Miniconda"
     - TARGET_ARCH: "x64"
       CONDA_PY: "27"
       CONDA_NPY: "18"
       PYTHON_VERSION: "2.7"
+      TEST_ALL_IMAGES: "yes" # only once, pick the one which made the most problems
```

Comment:
not sure. That was the one I had to do fixes for in this iteration. The last time it was a different version. I think that's probably just random?